capstone/src/main/webapp/script.js

```diff
@@ -31,14 +32,150 @@ async function submitInput() {
   if (!langElement) {
     return;
   }
+
+  // Get the selected attributes
   const attributes = document.getElementById("available-attributes").getElementsByTagName('input');
-  const requestedAttributes = []
+  const requestedAttributes = [];
   for (const attribute of attributes) {
     if (attribute.checked) {
       requestedAttributes.push(attribute.value);	
     }	
   }
-  await callPerspective(textElement.value, langElement.value, requestedAttributes);
+
+  // Get the selected analysis type
+  document.getElementById('analysis-container').innerHTML = '';
+  const radios = document.getElementsByName('analysisRadios');
+  var delimiter = "";
```

Comment:
(super) nit: Consider `tokenizer`. Delimiter usually refers to the character(s) that separate text (e.g. commas in CSVs). Tokenization refers to how text is split up:

https://www.geeksforgeeks.org/nlp-how-tokenizing-text-sentence-words-works/

Reply:
Fixed